guider/guider.py

```diff
@@ -10014,7 +10014,13 @@ def getPkg(name, isExit=True, isTemp=False, isRoot=True):
 
     @staticmethod
     def makeJsonString(dictObj):
-        return SystemManager.getPkg('json').dumps(dictObj, indent=2)
+        jsonString = SystemManager.getPkg('json').dumps(dictObj, indent=2)
```

Comment:
isn't it duplicated at 38581 line in reportSystemStat()?

> 
         # convert dict data to JSON data #
         jsonObj = SystemManager.makeJsonString(self.reportData)
         if not jsonObj:
             SystemManager.printWarning(\
                 "Fail to convert report data to JSON type")
             return
 
         # when encode flag is disabled, remove whitespace [\t\n\r\f\v] #
         if not SystemManager.encodeEnable:
             jsonObj = re.sub("\s", "", jsonObj) + "\n"
 
         # report system status to file #
         if SystemManager.reportObject:
             SystemManager.writeJsonObject(\
                 jsonObj, fd=SystemManager.reportObject, \
                 trunc=SystemManager.truncEnable)
>

Reply:
I'll update latest commit.
That would be fixed.